en/simulation/README.md

```diff
@@ -305,10 +296,24 @@ To route packets between SITL running on one computer (sending MAVLink traffic t
 More information about *mavlink-router* configuration can be found [here](https://github.com/intel/mavlink-router/#running).
 :::
 
+### Enable UDP Broadcasting
+
+The [mavlink module](../modules/modules_communication.md#mavlink_usage) routes to *localhost* by default, but you enable UDP broadcasting of heartbeats using its `-p` option.
+Any remote computer on the network can then connect to the simulator by listening to the appropriate port (i.e. 14550 for *QGroundControl*).
+
+:::note
+UDP broadcasting provides a simple way to set up the connection when there is only one simulation running on the network.
+Do not use this approach if there are multiple simulations running on the network (you might instead [publish to a specific computer](#enable-broadcast-to-specific-address)).
+:::
+
+This should be done in an appropriate configuration file where `mavlink start` is called.
+For example: [/ROMFS/px4fmu_common/init.d-posix/rcS](https://github.com/PX4/PX4-Autopilot/blob/master/ROMFS/px4fmu_common/init.d-posix/rcS).
+
 
-### Modify Configuration for External Broadcasting
+### Enable Broadcast to Specific Address
```

Comment:
Broadcast and Specific are contradicting. I'd write `Enable Streaming to Specific Address`

Reply:
Yes, that is MUCH better. Thanks. Fixed.